app/views/shared/_manage_school.html.erb

```diff
@@ -3,6 +3,7 @@
   <div class="dropdown-menu" aria-labelledby="manage_school" id="manage_school_menu">
     <%= link_to t('manage_school_menu.edit_school_details'), edit_school_path(school), class: 'dropdown-item' %>
     <%= link_to t('manage_school_menu.edit_school_times'), edit_school_times_path(school), class: 'dropdown-item' if can? :manage_school_times, school %>
+    <%= link_to t('manage_school_menu.your_school_estate'), edit_school_your_school_estate_path(school), class: 'dropdown-item' if EnergySparks::FeatureFlags.active?(:your_school_estates) && can?(:manage_alternative_heating_sources, school) %>
```

Comment:
You have a new ability here `manage_alternative_heating_sources` but do we need that if the heating source information is part of the school model? The check should just be whether the user can manage the school?

If an ability is used but not assigned then only Energy Sparks admins would ever see this feature.

Reply:
:+1:  have removed to match edit school (l4 - `edit_school_path(school)`)